server/policy_test.go

```diff
@@ -151,3 +166,113 @@ func TestEvaluatePolicyFailure(t *testing.T) {
 		})
 	}
 }
+
+func TestSevSnpSignedUefi(t *testing.T) {
+	// Generate fake signed reference measurements.
+	fw := ovmfsev.CleanExample(t, 0x200000)
+	meas, err := gcesev.LaunchDigest(gcesev.LaunchOptionsDefault(), fw)
+	if err != nil {
+		t.Fatal(err)
+	}
+	certdir := t.TempDir()
+	keydir := t.TempDir()
+	outdir := t.TempDir()
+	outdir2 := t.TempDir()
+	if err := devkeys.DumpTo(&devkeys.Options{
+		KeyDir:   keydir,
+		CertRoot: certdir,
+		CertDir:  "signer_certs",
+		Bucket:   "certs-dev",
+	}); err != nil {
+		t.Fatal(err)
+	}
+	fwdir := t.TempDir()
+	fwPath := path.Join(fwdir, "fw.fd")
+	if err := os.WriteFile(fwPath, fw, 0666); err != nil {
+		t.Fatal(err)
+	}
+
+	nonprod.RootCmd.SetArgs([]string{"endorse",
+		"--uefi", fwPath,
+		"--key_dir", keydir,
+		"--bucket_root", certdir,
+		"--root_path", "root.crt",
+		"--add_snp",
+		"--out_root", outdir,
+		"--snapshot_dir", "snap",
+	})
+	if err := nonprod.RootCmd.Execute(); err != nil {
+		t.Fatal(err)
+	}
+	dents, err := os.ReadDir(outdir)
+	if err != nil {
+		t.Fatal(err)
+	}
+	dents2, err := os.ReadDir(outdir2)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
I might be missing something, but I don't see where we use `outdir2`

Reply:
Removing..